### PR TITLE
Update 3DS Version in Debug and Appetize Podfiles

### PR DESCRIPTION
**Pull request by NQuinn27**
DEX-

# What this PR does

Updates the 3DS Version in both the Debug app and Appetize Apps podfiles

# Before merging

_QA_

- [x] I manually tested this with a demo application (simulator)
- [x] I manually tested this with a demo application (real device)
- [ ] I wrote unit tests for each new util-like function
- [ ] I wrote at least basic e2e tests for the new functionalities or new paths
- [ ] If this PR introduces UI changes, I manually tested this PR on iPhone 8 (iOS 10, 12, 14), iPhone 12, and iPhone 12 Max
- [ ] If this PR modifies the behavior of an API, I did my best to make my new API backward compatible
- [ ] If this PR modifies the API, I attempted to implement the feature with an example

_Documentation_

- [ ] If this PR modifies the API, I updated the API Reference.
- [ ] If this PR adds new options to the API, I wrote some documentation or guide in the online documentation
- [ ] If the PR modifies the API, I wrote some documentation to deprecate it

# After merging

- Make sure a new release has been pushed to Cocoapods trunk

### CODE REVIEW

**Review by jnewc — APPROVED** [review]
LGTM - but wondering why we have a separate + almost identical Podfile for Appetize

**Comment by NQuinn27** [comment]
@jnewc it looks to me that it is to do with the ipay88 pod failing on arm64. Perhaps appetize doesnt use the rosetta simulator? 
Either way, I think we can remove this soon - looks like the iPay88 library has been updated so we can build without rosetta.

File: Internal/Debug App/Appetize Podfile/Podfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '10.0'
+platform :ios, '13.0'
```
Comment (borisprimer) [inline Internal/Debug App/Appetize Podfile/Podfile:2]: The minimum should be 13.1 as stated in .podspec of the ThreeDS lib we are using.

**Review by borisprimer — CHANGES_REQUESTED** [review]

File: Internal/Debug App/Appetize Podfile/Podfile (hunk 1 above)
Comment (NQuinn27) [reply Internal/Debug App/Appetize Podfile/Podfile]: Updated, thanks!

**Review by borisprimer — APPROVED** [review]

**Pull request merged**